earth_enterprise/src/SConstruct

```diff
@@ -193,25 +193,19 @@ else:
   else:
     el_ver = 'el7'
 
-if bundle_3rd_libs:
-  python_version = '2.7.5'
+
+if el_ver == 'el6':
```

Comment:
I wonder if we need to consider el6 in the timeframe of this branch...

Reply:
I don't disagree. I just simplified the logic that was already there. When we drop python 2, we shouldnt need to do this sort of thing anymore, as everything should be running the same version of python.